### PR TITLE
:broom: Accommodate to knative-sandbox/reconciler-test#294

### PR DESCRIPTION
**Pull request by cardil**
## Proposed Changes

- :broom: Accommodate to knative-sandbox/reconciler-test#294

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -43,4 +43,5 @@ require (
 replace (
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v0.0.0-20210511163231-5b2d2b5f6c34
 	github.com/miekg/dns v1.0.14 => github.com/miekg/dns v1.1.25
+	knative.dev/reconciler-test => github.com/cardil/knative-reconciler-test v0.0.0-20220715181934-169264f8ba23
```
[the remark this answers is not in the document]
Comment (cardil) [reply go.mod]: Fixed in da8606eb00475b83bd14ac03fe48aad7e7454a8c

/unhold

**Comment by travis-minke-sap** [comment]
I'm kind of out of the picture these days and was hoping another code-owner would take a look, but am happy to un-block you.  I did a super quick review of the changes to make sure it wasn't malicious ; ) but haven't put any real thought into the changes.  

/approve
/lgtm

**Pull request merged**